core/migrations/0001_initial.py

```diff
@@ -1,8 +1,8 @@
 # Generated by Django 4.2.2 on 2023-07-09 09:43
 
+import django.db.models.deletion
```

Comment:
Migrations ideally shouldn't change so you can exclude them from the quality checks.